package.json

```diff
@@ -1,20 +1,23 @@
 {
   "name": "node-vault",
   "version": "0.7.1",
-  "description": "Javascript client for HashiCorp's Vault",
-  "main": "./src/index.js",
+  "description": "NodeJS API client for HashiCorp's Vault",
+  "main": "./src/main.js",
   "scripts": {
-    "precoverage": "npm run lint",
-    "coverage": "istanbul cover --report text-summary node_modules/.bin/_mocha",
-    "lint": "eslint .",
+    "lint": "standard",
```

Comment:
Would love to use the `--fix` flag as well.

Reply:
But the `--fix` flag doesn't *lint* the files, it *corrects* them. We could add a `lint-fix` task if you want, but I don't think it is necessary, as any dev can always run `npx standard --fix` to run `standard` without having it globally installed. If they *do* have it installed, they can simply run `standard --fix`.

Reply:
Also, they can always run `yarn lint --fix` as well.